map_view/display-layer-view-state/src/main/java/com/esri/samples/display_layer_view_state/DisplayLayerViewStateSample.java

```diff
@@ -16,132 +16,166 @@
 
 package com.esri.samples.display_layer_view_state;
 
+import java.util.ArrayList;
+import java.util.EnumSet;
+import java.util.List;
+
 import javafx.application.Application;
 import javafx.geometry.Insets;
 import javafx.geometry.Pos;
 import javafx.scene.Scene;
+import javafx.scene.control.Alert;
+import javafx.scene.control.Button;
 import javafx.scene.control.Label;
 import javafx.scene.layout.Background;
 import javafx.scene.layout.BackgroundFill;
 import javafx.scene.layout.CornerRadii;
 import javafx.scene.layout.StackPane;
 import javafx.scene.layout.VBox;
 import javafx.scene.paint.Paint;
+import javafx.scene.text.TextAlignment;
 import javafx.stage.Stage;
 
-import com.esri.arcgisruntime.data.ServiceFeatureTable;
+import com.esri.arcgisruntime.ArcGISRuntimeException;
 import com.esri.arcgisruntime.geometry.Point;
 import com.esri.arcgisruntime.geometry.SpatialReferences;
-import com.esri.arcgisruntime.layers.ArcGISMapImageLayer;
-import com.esri.arcgisruntime.layers.ArcGISTiledLayer;
 import com.esri.arcgisruntime.layers.FeatureLayer;
 import com.esri.arcgisruntime.layers.Layer;
 import com.esri.arcgisruntime.mapping.ArcGISMap;
+import com.esri.arcgisruntime.mapping.Basemap;
 import com.esri.arcgisruntime.mapping.Viewpoint;
+import com.esri.arcgisruntime.mapping.view.LayerViewStatus;
 import com.esri.arcgisruntime.mapping.view.MapView;
+import com.esri.arcgisruntime.portal.Portal;
+import com.esri.arcgisruntime.portal.PortalItem;
 
 public class DisplayLayerViewStateSample extends Application {
 
   private MapView mapView;
-
-  private static final int MIN_SCALE = 40000000;
-  private static final int TILED_LAYER = 0;
-  private static final int IMAGE_LAYER = 1;
-  private static final int FEATURE_LAYER = 2;
-
-  private static final String SERVICE_TIME_ZONES =
-      "https://sampleserver6.arcgisonline.com/arcgis/rest/services/WorldTimeZones/MapServer";
-  private static final String SERVICE_CENSUS =
-      "https://sampleserver6.arcgisonline.com/arcgis/rest/services/Census/MapServer";
-  private static final String SERVICE_RECREATION =
-      "https://sampleserver6.arcgisonline.com/arcgis/rest/services/Recreation/FeatureServer/0";
+  private FeatureLayer featureLayer;
+  private Label layerViewStatusLabel;
+  private Button toggleLayerButton;
```

Comment:
The label and button can be local variables, because they are only being used in the `start` method.